bootstorm-vm-boot-time/azuredeploy.json

```diff
@@ -0,0 +1,359 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.15.11.4",
+  "parameters": {
+    "azureAdApplicationId": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure AD Application ID to authenticate inside controller VM."
+      }
+    },
+    "azureAdApplicationPassword": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure AD Application password to authenticate inside controller VM."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "metadata": {
+        "description": "Azure Subscription Tenant Id."
+      }
+    },
+    "uniqueDnsNameForPublicIP": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+      }
+    },
+    "uniqueStorageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique Storage Account name to place OS disks."
+      }
+    },
+    "vmAdminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for the Virtual Machine."
+      }
+    },
+    "vmAdminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "vmCount": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of VMs to create."
+      }
+    },
+    "vmOsSku": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "allowedValues": [
+        "2012-R2-Datacenter",
+        "2012-Datacenter",
+        "2008-R2-SP1"
+      ],
+      "metadata": {
+        "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version. Allowed values: 2008-R2-SP1, 2012-Datacenter, 2012-R2-Datacenter."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "West US",
```

Comment:
Please remove default value of locations and let the user choose a location. Also please add more VM locations below and let the user choose all Azure locations.